interface/wx/html/htmprint.h

```diff
@@ -320,11 +320,12 @@ class wxHtmlEasyPrinting : public wxObject
 
     /**
         Set page footer. The following macros can be used inside it:
-         @@DATE@ is replaced by the current date in default format
-         @@PAGENUM@ is replaced by page number
-         @@PAGESCNT@ is replaced by total number of pages
-         @@TIME@ is replaced by the current time in default format
-         @@TITLE@ is replaced with the title of the document
+         - @@DATE@ is replaced by the current date in default format
+         - @@PAGENUM@ is replaced by page number
+         - @@PAGESCNT@ is replaced by total number of pages
+         - @@TIME@ is replaced by the current time in default format
+         - @@TITLE@ is replaced with the title of the document
+         - @@USER@@: is replaced with the user's name.
```

Comment:
Unless this is somehow by design
```suggestion
         - @@USER@: is replaced with the user's name.
```
Applies to all the occurrences of `@@USER@@` in the docs.

I am not sure how useful the information returned by the `wxUserName()` is for printouts, but I guess it is on the library user to decide if they want to use it.

BTW, `wxGetUserName()` , at least on Windows, is not a trivial function, is it?

Reply:
`wxGetUserName()` is a little "expensive", but header and footer macros are expanded just once in `OnPreparePrinting()`. So even if you are printing a thousand pages, it shouldn't be calling `wxGetUserName()` more than four times.

Personally, I find this really useful because I store these header/footer strings in a settings file, and it would be nice to have dynamic user names support.